Source/Common/gdcmByteSwap.txx

```diff
@@ -106,7 +106,7 @@ void Swap4(T &a, SwapCode const &swapcode)
 {
```

Comment:
When is `Swap4` meant to swap?  It seems to swap when the given `swapcode` *is opposite* the host endianness.

Reply:
@malaterre do you have an answer for this?